src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -250,4 +250,195 @@ public void testCreateGuestAccount(){
+        request.setSecurityQuestionKey("WHAT_WAS_THE_FIRST_CONCERT_YOU_ATTENDED");
+        request.setSecurityAnswer("Answer1");
+        request.setUidType("EMAIL");
+
+        response = new RestEngine().getResponse(map.get("gaCreateUrl"),headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.creation.Response responseElement =
+                response.as(com.training.pojos.ga.creation.Response.class);
+
+        Assertions.assertEquals(400,responseElement.getStatus(),ERROR_CODE);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getInternalMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getErrorCode()).as(ERROR_CODE)
+                .isEqualTo("GA-0101");
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getDeveloperMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getUserMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
+    }
+
+    @Test
+    public void testNegativeGaCreationInvalidPassword(){
+        String firstName="Audrey";
+        String lastName="Poole";
+        String email= "testemail@email.com";
+
+        com.training.pojos.ga.creation.Request request = new com.training.pojos.ga.creation.Request();
+        request.setBirthdate("19620802");
+        request.setEmail(email);
+        request.setFirstName(firstName);
+        request.setLastName(lastName);
+        request.setMarketingCountry("USA");
+        request.setPassword("Passwor");
+        request.setPrivacyAcceptDateTime("20190524T090712GMT");
+        request.setPrivacyVersion("1.11");
+        request.setTncAcceptDateTime("20190524T090712GMT");
+        request.setTncVersion("1.8");
+        request.setSecurityQuestion("What was the first concert you attended?");
+        request.setSecurityQuestionKey("WHAT_WAS_THE_FIRST_CONCERT_YOU_ATTENDED");
+        request.setSecurityAnswer("Answer1");
+        request.setUidType("EMAIL");
+
+        response = new RestEngine().getResponse(map.get("gaCreateUrl"),headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.creation.Response responseElement =
+                response.as(com.training.pojos.ga.creation.Response.class);
+
+        Assertions.assertEquals(422,responseElement.getStatus(),ERROR_CODE);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getInternalMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getErrorCode()).as(ERROR_CODE)
+                .isEqualTo("GA-0103");
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getDeveloperMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
```

Comment:
instead validating the length of the developer message, lets validate the actual developer message here.